data/forms/person-details.json

```diff
@@ -66,8 +66,17 @@
               "value": "F"
             },
             {
-              "label": "Prefer not to say",
-              "value": ""
+              "label": "Prefer to self-describe",
+              "value": "self"
+            }
+          ]
+        },
+        {
+          "name": "self-describe",
```

Comment:
This won't save back to the `gender` property on the person. I think we need a custom field for this, similar to how we've done date input.